libs/src/services/ethContracts/governanceClient.ts

```diff
@@ -164,36 +189,42 @@ class GovernanceClient extends ContractClient {
     return events.map(this.formatProposalEvent)
   }
 
-  async getProposalSubmission (
-    proposalId,
-    queryStartBlock = 0
-  ) {
+  async getProposalSubmission(proposalId: number, queryStartBlock = 0) {
```

Comment:
Technically, I believe proposalId should be a type BN throughout, but I don't think it'll ever get that big to cause an issue, so int should be fine

Reply:
If you want to know how I check these things, you can look at the ABI (Application Binary Interface) that's outputted by the contract compliation step. 
We import these ABI to know how to encode methods to send to the RPC node to communicate, but they have the "types" of all the input param and return types. 
ie. Looking here: uint256 is technically type BN, address is type string, bytes32 is type Uint8Array, (I'm fairly sure) etc. https://github.com/AudiusProject/audius-protocol/blob/82bac7496cd9a2ed79c77f064d3c54a5efe9c6f8/libs/eth-contracts/ABIs/GovernanceUpgraded.json#L389-L464

Reply:
ahh this is prefect thank you, was just doing best guess